repos/system_upgrade/common/libraries/overlaygen.py

```diff
@@ -405,15 +411,42 @@ def _create_mount_disk_image(disk_images_directory, path, disk_size):
     and it's supposed to be used for write directories of an overlayfs built
     above it.
 
+    If the disk_size is lower than 130 MiBs, the disk size is automatically
+    set to 130 MiBs to be able to format it correctly.
+
     The disk image is formatted with Ext4 if (envar) `LEAPP_OVL_IMG_FS_EXT4=1`.
 
     :param disk_images_directory: Path to the directory where disk images should be stored.
     :type disk_images_directory: str
     :param path: Path to the mountpoint of the original (host/source) partition/volume
     :type path: str
+    :param disk_size: Apparent size of the disk img in MiBs
+    :type disk_size: int
     :return: Path to the created disk image
     :rtype: str
     """
+    if disk_size < 130:
+        # NOTE(pstodulk): SEATBELT
+        # min. required size for current params to format a disk img with a FS:
+        #   XFS  -> 130 MiB
+        #   EXT4 -> 70  MiB
+        # so let's stick to 130 always. This is expected to happen when:
+        #  * the free space on a system mountpoint is really super small, but if
+        #    such a mounpoint contains a content installed by packages, most
+        #    likely the msg about not enough free space is raised
+        #  * the mountpoint is actually no important at all, could be possibly
+        #    read only (e.g. ISO), or it's an FS type that should be covered by
+        #    OVERLAY_DO_NOT_MOUNT
+        #  * most common case important for us here could be /boot, but that's
+        #    covered already in different actors/checks, so it should not be
+        #    problem either
+        # NOTE(pstodulk): In case the formatting params are modified,
+        # the minimal required size could be different
+        api.current_logger().warning(
+            'The apparent size for the disk image representing {path}'
+            ' is too small ({} MiBs) for a formatting. Setting 130 MiBs instead.'
```

Comment:
This warning does not have its placeholders formatted (a missing `.format` perhaps)